android/src/main/java/com/dylanvann/fastimage/FastImagePreloaderModule.java

```diff
@@ -4,18 +4,20 @@
 
 import com.bumptech.glide.Glide;
 import com.bumptech.glide.load.model.GlideUrl;
+import com.facebook.react.bridge.Promise;
 import com.facebook.react.bridge.ReactApplicationContext;
 import com.facebook.react.bridge.ReactContextBaseJavaModule;
 import com.facebook.react.bridge.ReactMethod;
 import com.facebook.react.bridge.ReadableArray;
 import com.facebook.react.bridge.ReadableMap;
 import com.facebook.react.views.imagehelper.ImageSource;
 
-class FastImageViewModule extends ReactContextBaseJavaModule {
+class FastImagePreloaderModule extends ReactContextBaseJavaModule {
```

Comment:
Why rename class just like that. I have #425 which edit FastImageViewModule.java